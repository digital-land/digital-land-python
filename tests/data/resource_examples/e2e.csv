schema-three,name,organisation,adress,amount,date
id1,A library,local-authority-eng:AAA,123 avenue,90,11/10/2018
id2,Leisure centre,,456 road,680,
id3,,local-authority-eng:AAA,Town Hall London,34,06/09/2014
,Park,local-authority-eng:AAA,N/A,0.4,01/04/2020
