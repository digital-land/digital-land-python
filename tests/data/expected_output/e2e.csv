schema-three,name,organisation,address,amount,date,slug
id1,A library,local-authority-eng:AAA,123 avenue,90,2018-10-11,/schema-three/local-authority-eng/AAA/id1
id2,Leisure centre,,456 road,680,,
id3,Town Hall London,local-authority-eng:AAA,Town Hall London,34,2014-09-06,/schema-three/local-authority-eng/AAA/id3
,Park,local-authority-eng:AAA,none,0.4,2020-04-01,
